provision.sh

```diff
@@ -249,7 +249,7 @@ fi
 # Enable the rosetta theme
 # To see the available themes, execute: npm run wp-env run cli wp theme list
 print_header "Enabling the wporg theme"
-$WP_CLI_PREFIX wp theme activate pub $WP_CLI_SUFFIX
+$WP_CLI_PREFIX wp theme activate pub/wporg $WP_CLI_SUFFIX
```

Comment:
I think this needs to be just `pub`, this change was introduced in https://github.com/WordPress/translate-w-org-env/commit/e0ca227009f4fb67ecf6ee1298958da3328042ee.

If you do `wp theme list`, the theme is named `pub`, I believe,

Reply:
Oh, then this is a difference between LAMP and wp-env. Maybe we can get them the same via changing the mapping

Reply:
`pub/wporg` is closer to what the real WordPress.org structure is

Reply:
Ok, that makes sense, I'll look into it.

Reply:
I've pushed the necessary change, could you try if it still works in wp-env?

Reply:
I realized I had already merge it on Github, so I pushed the changes to the branch and re-merged it locally. So the change is already in main, committed via https://github.com/WordPress/translate-w-org-env/commit/af0f92401171f8b0333cdc93b0e8803dd38ac94b

Reply:
It works, thanks!